app/javascript/angular/code/controllers/_sessions_list_ctrl.js

```diff
@@ -110,23 +110,26 @@ export const SessionsListCtrl = (
   );
 
   $scope.$on("markerSelected", function(event, data) {
-    $scope.toggleSession(data.session_id, true);
+    $scope.toggleSession(
+      data.session_id,
+      elmApp.ports.toggleSessionSelection.send
+    );
     $scope.$apply();
   });
 
-  $scope.toggleSession = function(sessionId, markerSelected) {
+  $scope.toggleSession = function(sessionId, callback) {
     if (this.isSessionDisabled(sessionId)) {
```

Comment:
do we still need that if we have radio buttons now?

Reply:
Good catch. Prolly not but will leave this to another round of removals